errors/next-prerender-random-client.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Cannot access `Math.random()` from a Client Component without a fallback UI defined
+---
+
+## Why This Error Occurred
+
+A Client Component is calling `Math.random()`.
+
+Client Components primarily run in the browser however on an initial page visit, Next.js will server an HTML page produced by simulating the client environment on the server. This process is called Server Side Rendering or SSR. Next.js will attempt to prerender this HTML ahead of time however if a Client Component accesses a random source during this prerender, it cannot include this component in the prerendered HTML, otherwise the value will be fixed on each user request and not random like expected. Next.js will use the nearest Suspense boundary around this component to prerender a fallback instead however in this instance there was no Suspense boundary.
```

Comment:
```suggestion
Client Components primarily run in the browser however on an initial page visit, Next.js will serve an HTML page produced by simulating the client environment on the server. This process is called Server Side Rendering or SSR. Next.js will attempt to prerender this HTML ahead of time, however if a Client Component accesses a random source during this prerender, it cannot include this component in the prerendered HTML, otherwise the value will be fixed on each user request and not random like expected. Next.js will use the nearest Suspense boundary around this component to prerender a fallback instead however in this instance there was no Suspense boundary.
```